src/doc/rustc/src/platform-support/wasm32-wasip1.md

```diff
@@ -132,3 +132,10 @@ It's recommended to conditionally compile code for this target with:
 
 Note that the `target_env = "p1"` condition first appeared in Rust 1.80. Prior
 to Rust 1.80 the `target_env` condition was not set.
+
+## Enabled WebAssembly features
+
+The default set of WebAssembly features enabled for compilation is currently the
+same across all WebAssembly targets. For more information on WebAssembly
```

Comment:
The wasm32-wasip1-threads target also enables bulk-memory and atomics, right?

Reply:
Indeed!